zerver/lib/test_classes.py

```diff
@@ -559,6 +560,13 @@ def assert_not_in_success_response(self, substrings: List[str],
         for substring in substrings:
             self.assertNotIn(substring, decoded)
 
+    def assert_logged_in_user_id(self, user_id: Optional[int]=None) -> None:
```

Comment:
Let's make this a positional argument that can be None, not a kwarg (see comment below for why)